### DISCUSSION TITLE
<XFlowCanvas>画布节点之间的连线 如何变成直角

### DISCUSSION THREAD

**Discussion by h5-Old-driver** [antvis/X6#2205] [Q&A] [answered]
我的项目中只需要展示流程图  使用的是快速上手中的  <XFlowCanvas> 画布 
节点之间的连线看起来像是乱麻  怎么才能实现 <FlowchartCanvas>  中的那种 直线的连接线呢
下面是图片  
![image](https://user-images.githubusercontent.com/81205631/172844053-c0c5765d-fb42-4c92-a318-8daeb83a49de.png)

**Comment by NewByVector** [comment, marked as answer]
可以到 [XFlow](https://github.com/antvis/XFlow) 下面提交 issue.
